txpipe/noise_maps.py

```diff
@@ -619,6 +621,7 @@ def run(self):
             # The top section has the metadata in it
             group = outfile.file.create_group("maps")
             group.attrs["nbin_source"] = nbin_source
```

Comment:
is this line not redundant because of the line below?

Reply:
I left a few things like this in as a backup in case there was old code that needed it.

Reply:
I'll put in a TODO to fix it.